backend/src/components/realmAdminSvc.js

```diff
@@ -388,6 +387,23 @@ class RealmAdminService {
     //204 created...
     return response;
   }
+
+  // Get from the users list with the username query param. If we want to be able to filter by more can refactor this method
+  // to take in search parameters. https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_users_resource
```

Comment:
yeah, we can think more on this later; this is a good one for our immediate needs.
totally possible to call with the search parameter that will query against username, first or last name, or email.  We would then need to iterate over the results to ensure that there is a correct match from idir.  Would be good for accounts like mine that are initially done via github, then have an idir account associated with them.  But let's leave that when we build out a more robust admin interface.

`https://sso-dev.pathfinder.gov.bc.ca/auth/admin/realms/jbd6rnxw/users?briefRepresentation=false&first=0&max=100&search=sherman`

Then look in the federatedIdentities / identityProvider = idir / username =

Reply:
Sounds good